src/ini-load.js

```diff
@@ -31,14 +32,43 @@ const EXCLUDE_INI_LOAD = [
  * @param {!Element|!./service/ampdoc-impl.AmpDoc} elementOrAmpDoc
  * @param {!Window} hostWin
  * @param {!./layout-rect.LayoutRectDef} rect
- * @param {boolean=} opt_isInPrerender signifies if we are in prerender mode.
+ * @param {boolean=} opt_prerenderableOnly signifies if we are in prerender mode.
```

Comment:
why the name change? I'm not sure what `opt_prerenderableOnly` means

Reply:
I believe "is in prerender" was a bad name, tbh. It never meant that this method is called in "prerender mode". All it meant is that - "only prerenderable components should be considered for this computation".